docs/reference/changelog-r2021.md

```diff
@@ -3,6 +3,8 @@
 ## Webots R2021c
 Released on XX, XXth, 2021.
 
+  - New Features:
+    - Released experimental [Skin](skin.md) node ([#3566](https://github.com/cyberbotics/webots/pull/3566)).
```

Comment:
... and I would remove experimental here as it is not experimental any more after this PR.
```suggestion
    - Released [Skin](skin.md) node ([#3566](https://github.com/cyberbotics/webots/pull/3566)).
```